src/transforms/rotate.jl

```diff
@@ -6,15 +6,15 @@
     Rotate(rot)
 
 Rotate geometry or mesh with rotation `rot`
-from ReferenceFrameRotations.jl. 
+from Rotations.jl.
 
 ## Examples
 
 ```julia
-Rotate(EulerAngleAxis(pi/4, [1, 0, 0]))
+Rotate(rand(QuatRotation{Float64}))
```

Comment:
I added an example with `QuatRotation <: Rotation{3}` which represents a rotation with a quaternion.

Reply:
Is there an example with a more familiar rotation type? Maybe `AngleAxis`? Quaternions are a bit exoteric for non-mathematicians.